systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -1069,6 +1077,453 @@ void testRestoreOffsetsInConsumerGroup(ExtensionContext extensionContext) {
+            .withKafkaUsername(userTarget.getMetadata().getName())
+            .withConsumerGroupName(ClientUtils.generateRandomConsumerGroup())
+            .build();
+
+        LOGGER.info("Consumer in target cluster and topic should receive {} messages", messagesCount);
+        internalKafkaClient.checkProducedAndConsumedMessages(sent, internalKafkaClient.receiveMessagesTls());
+        LOGGER.info("Messages successfully mirrored");
+
+        LabelSelector zkSourceSelector = KafkaResource.getLabelSelector(kafkaClusterSourceName, KafkaResources.zookeeperStatefulSetName(kafkaClusterSourceName));
+        LabelSelector kafkaSourceSelector = KafkaResource.getLabelSelector(kafkaClusterSourceName, KafkaResources.kafkaStatefulSetName(kafkaClusterSourceName));
+        LabelSelector zkTargetSelector = KafkaResource.getLabelSelector(kafkaClusterTargetName, KafkaResources.zookeeperStatefulSetName(kafkaClusterTargetName));
+        LabelSelector kafkaTargetSelector = KafkaResource.getLabelSelector(kafkaClusterTargetName, KafkaResources.kafkaStatefulSetName(kafkaClusterTargetName));
+
+        Map<String, String> kafkaSourcePods = PodUtils.podSnapshot(namespaceName, kafkaSourceSelector);
+        Map<String, String> zkSourcePods = PodUtils.podSnapshot(namespaceName, zkSourceSelector);
+        Map<String, String> kafkaTargetPods = PodUtils.podSnapshot(namespaceName, kafkaTargetSelector);
+        Map<String, String> zkTargetPods = PodUtils.podSnapshot(namespaceName, zkTargetSelector);
+
+        LOGGER.info("Change CA cert for Source clusters by forcing annotation");
+        String sourceClusterCaSecretName = KafkaResources.clusterCaCertificateSecretName(kafkaClusterSourceName);
+        SecretUtils.annotateSecret(namespaceName, sourceClusterCaSecretName, Ca.ANNO_STRIMZI_IO_FORCE_RENEW, "true");
+        zkSourcePods = RollingUpdateUtils.waitTillComponentHasRolledAndPodsReady(namespaceName, zkSourceSelector, 3, zkSourcePods);
+        kafkaSourcePods = RollingUpdateUtils.waitTillComponentHasRolledAndPodsReady(namespaceName, kafkaSourceSelector, 3, kafkaSourcePods);
+        mmSnapshot = DeploymentUtils.waitTillDepHasRolled(namespaceName, KafkaMirrorMaker2Resources.deploymentName(clusterName), 1, mmSnapshot);
+
+        LOGGER.info("Change CA cert for Target clusters by forcing annotation");
+        String targetClusterCaSecretName = KafkaResources.clusterCaCertificateSecretName(kafkaClusterTargetName);
+        SecretUtils.annotateSecret(namespaceName, targetClusterCaSecretName, Ca.ANNO_STRIMZI_IO_FORCE_RENEW, "true");
+        zkTargetPods = RollingUpdateUtils.waitTillComponentHasRolledAndPodsReady(namespaceName, zkTargetSelector, 3, zkTargetPods);
+        kafkaTargetPods = RollingUpdateUtils.waitTillComponentHasRolledAndPodsReady(namespaceName, kafkaTargetSelector, 3, kafkaTargetPods);
+        DeploymentUtils.waitTillDepHasRolled(namespaceName, KafkaMirrorMaker2Resources.deploymentName(clusterName), 1, mmSnapshot);
```

Comment:
`KafkaMirrorMaker2Resources.deploymentName(clusterName)` to variable and use it in whole test.